src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Microsoft.Extensions.Configuration.AzureAppConfiguration.csproj

```diff
@@ -17,17 +17,10 @@
     <PackageReference Include="Azure.Data.AppConfiguration" Version="1.3.0-beta.3" />
     <PackageReference Include="Azure.Messaging.EventGrid" Version="4.7.0" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.3.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.18" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.18" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
     <PackageReference Include="System.Text.Json" Version="4.7.2" />
```

Comment:
Is System.Text.Json required in netstandard2.1?

Reply:
No, will pull it out to a conditional item group

Reply:
Actually, wondering if it's required to be explicitly referenced at all. I think the SDK uses it.

Reply:
It's included [here](https://github.com/Azure/azure-sdk-for-net/blob/ddebc42fa17e83daa317481b9844a606533b9d9b/sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj) but doesn't specify the version.

Reply:
sounds like we don't' need to reference it then. Take a look on nuget, System.Text.Json shows up as a dependency.